src/components/homePage/index.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react'
+import { NavLink } from 'react-router-dom'
+import backgroundLogo from '../assets/bg_logo.svg'
+import logo from '../assets/logo.svg'
+import styled from '@emotion/styled'
+
+const HomePage = () => {
+  return (
+    <$HomePageContainer>
+      <$LeftContainer>
+        <header>
+          <NavLink to="#">about us</NavLink>
+
+          <NavLink to="#">our raids</NavLink>
+
+          <NavLink to="#">our stats</NavLink>
+
+          <NavLink to="#">contact us</NavLink>
+        </header>
+      </$LeftContainer>
+      <$RightContainer>
+        <$BackgroundOverlayContainer>
+          <div>
+            <img src={logo} alt="OS Raid Guild" />
+          </div>
+        </$BackgroundOverlayContainer>
+        <$BackgroundContainer />
+      </$RightContainer>
+    </$HomePageContainer>
+  )
+}
+
+const $HomePageContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+`
+
+const $LeftContainer = styled.section`
+  flex-grow: 1;
+  padding-top: 2rem;
+
+  header {
```

Comment:
I think `nav` might be more appropriate than `header` here, what do you think?

Reply:
Yes you're right. I've updated this in next commit. Thanks for the suggestion.